.Lib9c.Tests/Model/RankingSimulatorTest.cs

```diff
@@ -167,5 +167,71 @@ public void ConstructorWithCostume()
             var enemyPlayer = (EnemyPlayer)e.Character;
             Assert.Equal(row2.Stat, enemyPlayer.Stats.OptionalStats.DEF);
         }
+
+        [Theory]
+        [InlineData(1, 1000)]
+        [InlineData(55, 1000)]
+        [InlineData(85, 1000)]
+        [InlineData(120, 1000)]
+        [InlineData(160, 1000)]
+        [InlineData(200, 1000)]
+        public void CheckToReceiveAllRewardItems(int level, int simulationCount)
```

Comment:
In my opinion, it would be better to check the order of randomly selected values. (like [WeightedSelectorTest.SelectV2Multiple](https://github.com/planetarium/lib9c/blob/1e3912bd78b634a1e2760c30f291a325a08a8c44/.Lib9c.Tests/Model/WeightedSelectorTest.cs#L65))

Reply:
That's a good idea. I Will add test case.